lib/npm.js

```diff
@@ -124,7 +124,7 @@
       npm.command = c
       if (commandCache[a]) return commandCache[a]
 
-      var cmd = require(path.join(__dirname, a + '.js'))
+      var cmd = require("./" + path.join(__dirname, a) + ".js");
```

Comment:
This needs to be `''` because `standard`.

Reply:
Oh and this needs to be `'./' + a + '.js'`. Drop the __dirname.

Reply:
perfect!! Will change.